### PR TITLE
[14.0][MIG] hr_payroll_account_operating_unit

### PR DESCRIPTION
**Pull request by JuanyDForgeflow**
Migration to 14.0

This module introduces the following features:

* Adds Operating Unit (OU) to the account moves and its lines created by the payslip, based on the Operating Unit (OU) defined in the Employee's Contract.

### CODE REVIEW

File: hr_payroll_account_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#   (http://www.eficent.com)
+# Copyright 2016-17 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "HR Payroll Account Operating Unit",
+    "version": "14.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Eficent Business and IT Consulting Services S.L., "
```
Comment (AaronHForgeFlow) [inline hr_payroll_account_operating_unit/__manifest__.py]:
```suggestion
    "author": "ForgeFlow S.L., "
```

Can you do this also in the Copyrights? You can use a separate commit. Thank you!

Comment (JuanyDForgeflow) [reply hr_payroll_account_operating_unit/__manifest__.py]: @AaronHForgeFlow It's done now

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests

**Pull request merged**